UnityProject/Assets/Scripts/Messages/MessageTypes.cs

```diff
@@ -15,11 +15,13 @@ enum MessageTypes : short
     UpdateUIMessage = 1006,
     ClosetHandlerMessage = 1007,
     ForceJobListUpdateMessage = 1008,
+    TransformStateMessage = 1008,
```

Comment:
shouldn't this be 1009?

Reply:
Wow, good catch! I'm surprised it worked all the time

Reply:
well it does explain the error on first join maybe?

Reply:
It most likely does, will fix